### PR TITLE
authz queries to meet the expanded types provided in the sdk

### PR DESCRIPTION
**Pull request by mvid**

### CODE REVIEW

**Comment by mvid** [comment]
This PR is split from https://github.com/cosmos/cosmjs/pull/1307

**Review by webmaster128 — COMMENTED** [review]
Diff looks good to me, thank you!

Could you add tests for the new queries to `packages/stargate/src/modules/authz/queries.spec.ts`? There you already see a test for the `grants` query which I guess can easily be copied and adapted.

**Comment by webmaster128** [comment]
I re-opened to try to trigger a CI run. Not sure what kind of problem CircleCI has here. Maybe a few more commits are sufficient, let's see.

**Comment by mvid** [comment]
@webmaster128 Took some tries to get the linter/tests to work, since I am in Intellij which isn't well supported in this setup, but I believe this is now ready!

**Comment by mvid** [comment]
@webmaster128 Any more notes?

**Comment by mvid** [comment]
Pinging this for an update

File: packages/stargate/src/modules/authz/queries.spec.ts [hunk 1]
```diff
@@ -95,7 +96,59 @@ describe("AuthzExtension", () => {
...
 
-      // Check if its the same one then we granted
+      // Check if it's the same one then we granted
+      expect(msgDecoded).toEqual(grantedMsg);
+
+      tmClient.disconnect();
+    });
+
+    it("works querying by granter", async () => {
+      pendingWithoutSimapp46();
```
Comment (webmaster128) [inline packages/stargate/src/modules/authz/queries.spec.ts:106]: Added in cosmos-sdk 0.45.2 👍

**Review by webmaster128 — COMMENTED** [review]
Nice, thank you.

Just one point: could you make one `describe` block per method? I.e. add the tests on the same level as `describe("grants", () => {`. This allows us to add multiple tests per method (not needed here).

**Comment by mvid** [comment]
@webmaster128 done

**Comment by webmaster128** [comment]
Nice, thank you! Could you enable modification for maintainers? I rebased this to latest main and added a CHANGELOG entry to get it merged.

**Comment by mvid** [comment]
@webmaster128 That is odd... I already have it set to allow edits by maintainers

Screenshot 2023-01-03 at 15 36 43

**Comment by webmaster128** [comment]
Hmm, very strange. Don't know. In this case could you rebase to latest main (or merge main here) and add this CHANGELOG entry to the Unreleased secition?

```
### Added

- @cosmjs/stargate: Add `granteeGrants` and `granterGrants` queries to
  `AuthzExtension` ([#1308]).

[#1308]: https://github.com/cosmos/cosmjs/pull/1308
```

**Comment by mvid** [comment]
Done, pulled main and added the log entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -65,10 +65,13 @@ and this project adheres to
   `HttpBatchClientOptions`, `RpcClient` ([#1311]).
 - @cosmjs/tendermint-rpc: Send batch immediately when full in `HttpBatchClient`
   ([#1310]).
+- @cosmjs/stargate: Add `granteeGrants` and `granterGrants` queries to
+  `AuthzExtension` ([#1308]).
 
 [#1309]: https://github.com/cosmos/cosmjs/issues/1309
 [#1310]: https://github.com/cosmos/cosmjs/issues/1310
 [#1311]: https://github.com/cosmos/cosmjs/issues/1311
+[#1308]: https://github.com/cosmos/cosmjs/pull/1308
```
Comment (webmaster128) [inline CHANGELOG.md:74]: This was now added to an old release. It needs to move up ti the `## [Unreleased]` section for the upcoming 0.30.0

Comment (mvid) [reply CHANGELOG.md]: My mistake. Pushed an update

**Review by webmaster128 — APPROVED** [review]
Thank you!

**Pull request merged**